guide/12-enrich-data-with-thematic-information/part1_introduction_to_geoenrichment.ipynb

```diff
@@ -11,139 +11,576 @@
     "# Part 1 - Introduction to GeoEnrichment"
```

Comment:
<p>Facility 1 and facility 2 have higher populations, are diverse, and have lower income. Facility 3 is far younger, and has larger households, less education, and lower incomes. Facility 4 and facility 5 are older, more educated, and have higher incomes. </p><p> </p><p>If interested in opening a discount department store, facility 2 is the most attractive location, with facility 1 as a close second. The diversity and lower income can allow us to conclude that people will buy products at lower prices. </p><p> </p><p>If interested in opening a quick service restaurant, facility 3 may be the best option to meet the needs of a young, busy, and price conscious population. </p><p> </p><p>Obviously, depending on the key characteristics of the business looking for a new location, the key demographic indicators will be different. Using geoenrichment with the ArcGIS API for Python enables extremely quick access to demographic variables for informed decision making.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>